foundation-extensions/mobile-core/identity/acpidentity-aepidentity.md

```diff
@@ -0,0 +1,224 @@
+# Migrating to AEPIdentity reference
+
+This document is a reference comparison of AEPIdentity \(3.x\) APIs against their equivalent APIs in ACPIdentity \(2.x\) for an iOS mobile application implementation.
+
+## Public Classes
+
+| Type | AEP 3.x \(Swift\) | AEP 3.x \(Objective-C\) | ACP 2.x \(Objective-C\) |
+| :--- | :--- | :--- | :--- |
+| Primary Class | Identity | AEPMobileIdentity | ACPIdentity |
+| Class | MobileCore | AEPMobileCore | ACPCore |
+
+## Identity extension APIs
+
+For more information, please read the [Identity API reference](identity-api-reference.md).
+
+### appendVisitorInfoForURL
+
+{% tabs %}
+{% tab title="AEP 3.x \(Swift\)" %}
+
+```swift
+Identity.appendTo(url:completion:)
```

Comment:
As for other extensions, we should have below for the APIs in this page
static func appendTo(url: URL?, completion: @escaping (URL?, Error?) -> Void)